code_samples/back_office/limitation/translations/forms.en.yaml

```diff
@@ -0,0 +1,4 @@
+role.policy.custom_module: 'Custom module'
+role.policy.custom_module.all_functions: 'Custom module / All functions'
+role.policy.custom_module.custom_function_1: 'Custom module / Function #1'
+role.policy.custom_module.custom_function_2: 'Custom module / Function #2'
```

Comment:
TODO: Study #2205 for an example allowing `translation:extract` to generate this file (something like `php bin/console translation:extract en --domain=forms --dir=src --output-dir=translations`)